flagengine/utils/hashing.go

```diff
@@ -35,3 +35,7 @@ var hashedPercentageForObjectIdsFunc = getHashedPercentageForObjectIds
 func MockSetHashedPercentageForObjectIds(fn func([]string, int) float64) {
 	hashedPercentageForObjectIdsFunc = fn
 }
+
+func ResetMocks() {
```

Comment:
Perhaps this is a hangover from a previous attempt at mocking? I can't see that this is being used anywhere?

Reply:
https://github.com/Flagsmith/flagsmith-go-client/pull/88#diff-e4c09beef6da84b10fa7aeaba7ceb03078ba6215f35c0ab95030d83a62fa5c70R300

Reply:
It's used here:  https://github.com/Flagsmith/flagsmith-go-client/blob/6ffd9c634841afae45ff6edbdccd8799351e2689/flagengine/segments/evaluator_test.go#L300

The previous code that was supposed to reset the function worked incorrectly, causing an infinite recursion.